packages/bpk-component-tooltip/src/BpkTooltipPortal.js

```diff
@@ -169,13 +176,18 @@ class BpkTooltipPortal extends Component<Props, State> {
     const classNames = [getClassName('bpk-tooltip-portal')];
     const renderPortal = !hasTouchSupport() || !hideOnTouchDevices;
 
+    const targetWithAccessibilityProps = React.cloneElement(target, {
+      tabIndex: '0',
+      'aria-label': ariaLabel,
```

Comment:
What happens if the target a consumer supplies already has an aria-label set? Do we worry that it would then get overriden by this one?

e.g.

```
  <BpkTooltip
    ariaLabel="London Heathrow"
    id="my-tooltip"
    target={<BpkText ariaLabel="London Heathrow Airport" textStyle="lg">LHR</BpkText>}
  >
  >
    London Heathrow
  </BpkTooltip>
```

Maybe we need to make sure its clear in the docs around how the aria-label would be used so that if someone does the above scenario they know they should be passing it through the prop rather than just supply it straight to the target?

Reply:
I know its rare but best to think about it and cover our bases just in case?

Reply:
Yeah it would override it, that's a good point. I'll add a note to the docs now

Reply:
Maybe in the changeling we should say "if you're already providing an ariaLabel to the target, ensure you move this to the tooltip"?